authzed/spicedb
# chore: add github issue templates

## Code Review Request

- Verify the labels aligned
- Verify if you only want those templates. I noticed you different "bug" related labels (which you could apply later) sometimes they are helpful, but it is tricky to add them based on the input (so they can not be optional type of thing)
- Any other link?
- Open to add https://github.com/marketplace/actions/advanced-issue-labeler for dynamic labels based on the checkboxes?!

## Intent

- Help members outside the authzed organization to contribute to the project by directing them to the right places
- Help project maintainers to have more structured information and labeling

## How-To Test

- Visit https://github.com/yordis/spicedb/issues/new/choose
- Verify you are happy with it
